Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -356,14 +410,24 @@ attribute [local simp] SSet.standardSimplex
 /-- The functor which sends `[n]` to the simplicial set `Δ[n]` equipped by
 the obvious augmentation towards the terminal object of the category of sets. -/
 @[simps]
-noncomputable def standardSimplex : SimplexCategory ⥤ SSet.Augmented where
+noncomputable def standardSimplex : SimplexCategory ⥤ SSet.Augmented.{u} where
   obj Δ :=
     { left := SSet.standardSimplex.obj Δ
       right := terminal _
       hom := { app := fun Δ' => terminal.from _ } }
   map θ :=
     { left := SSet.standardSimplex.map θ
       right := terminal.from _ }
+  map_id n := by
+    ext1
+    · dsimp
+      erw [yoneda.map_id, uliftFunctor.map_id]
```

Comment:
Can we teach `aesop_cat` to take care of these proofs?

Reply:
Yes, I have just improved automation.